README.md

```diff
@@ -120,15 +120,18 @@ A few basic steps are needed to get set up:
    5. Install build dependencies (Arch packages are listed here, your distribution will likely be similar):
 	  - `clang`: required by RocksDB.
 	  - `protobuf`: required for protobuf serialization (gRPC).
-	  - `cmake`: required for building protobuf
+	  - `cmake`: required for building protobuf.
+	  - `git-lfs`: The Git extension for [Large File Support](https://git-lfs.github.com/) (required for EF tests submodule).
    6. Navigate to the working directory.
-   7. Run the test by using command `cargo test --all`. By running, it will pass all the required test cases.
+   7. If you haven't already, clone the repository with submodules: `git clone --recursive https://github.com/sigp/lighthouse`.
+		Alternatively, run `git submodule init` in a repository which was cloned without submodules.
+   8. Run the test by using command `cargo test --all`. By running, it will pass all the required test cases.
```

Comment:
I think we should make this command `cargo test --all --release`.

The tests run much, much faster when compiled to release :)

Reply:
Sounds good to have a faster sanity check when compiling for the first time.

Reply:
Just checking, do you intend to add the `--release` flag?

Reply:
Done